packages/markdownit/plugin.js

```diff
@@ -1,5 +1,12 @@
 import MarkdownIt from 'markdown-it'
 
+function _requireCjs(module) {
+  if (module.__esModule) {
```

Comment:
Can we simply check for `default` prop existence and pick it? It's the usual behavior for bundlers  for CJS fallback

Reply:
Hi @pi0, thanks for the feedback !
Checking for module.__esModule seems to be the usual behavior for CJS fallback.
For example this is what is used in babel : https://github.com/babel/babel/search?q=%22function+_interopRequireDefault%22&unscoped_q=%22function+_interopRequireDefault%22

Reply:
What if a CJS module does something like
```
exports = myPlugin()
exports.default = defaultHandler()
```

In this (edge) case, we want to require `myPlugin()` but if we check for `default` existence to determine the module type, we would require `defaultHandler`.

Reply:
True for babel and correct way. However, there is/was not a widely adopted standard to use `__esModule` and there may be conditions that do not export this getter. At least rollup does this:

```js
function _interopDefault (ex) { return (ex && (typeof ex === 'object') && 'default' in ex) ? ex['default'] : ex; }
```

And also we do a similar thing on nuxt core. So it makes sense for consistency everywhere.

Reply:
Okay, thanks for the feedback, I updated the PR :)